energysmartbridge/src/MQTT/BinarySensor.cs

```diff
@@ -10,6 +10,7 @@ public enum DeviceClass
         {
             heat,
             problem,
+            moisture,
```

Comment:
Are you sure we should be using moisture for the leak detected? Since leak detected is just a boolean, and the HA docs say moisture is for a percentage, I am not sure this will map or display properly in the frontend